third_party/openthread/CMakeLists.txt

```
@@ -37,6 +37,7 @@ set(OT_JOINER ON CACHE BOOL "enable joiner" FORCE)
 set(OT_LEGACY ON CACHE STRING "enable legacy network support" FORCE)
 set(OT_SLAAC ON CACHE BOOL "enable SLAAC" FORCE)
 set(OT_TREL ON CACHE BOOL "enable TREL")
+set(OT_LINK_METRICS_SUBJECT ON CACHE BOOL "enable link metrics subject" FORCE)
```

nit: sort alphabetically
```suggestion
set(OT_LEGACY ON CACHE STRING "enable legacy network support" FORCE)
set(OT_LINK_METRICS_SUBJECT ON CACHE BOOL "enable link metrics subject" FORCE)
set(OT_SLAAC ON CACHE BOOL "enable SLAAC" FORCE)
set(OT_TREL ON CACHE BOOL "enable TREL")
```